apps/wpcom-block-editor/src/calypso/features/iframe-bridge-server.js

```diff
@@ -697,13 +700,36 @@ async function openLinksInParentFrame( calypsoPort ) {
 		'.interface-interface-skeleton__sidebar, .interface-interface-skeleton__secondary-sidebar'
 	);
 	for ( const sidebar of sidebars ) {
-		observeSidebarMutations( sidebar );
+		handleSidebarNodeAdded( sidebar );
 	}
 
 	// Add and remove the sidebar observers as the sidebar elements appear and disappear.
 	// They are always direct children of the body element.
-	const body = document.querySelector( '.interface-interface-skeleton__body' );
-	sidebarsObserver.observe( body, { childList: true } );
+	const sidebarBodyElem = document.querySelector( '.interface-interface-skeleton__body' );
+	sidebarsObserver.observe( sidebarBodyElem, { childList: true } );
```

Comment:
The changes from L623-L709 appear to just rename the variables. IMO, the old names aren't necessarily worse. Since they're subjective, could we leave these changes out of the PR to avoid marking these lines as changed in the commit? (This way, it's easier to track down the commit where the main changes to those lines happened.)

Reply:
I would also vote for leaving the name changes out from this PR. This would make the PR focused on that one bug fix.

Reply:
@noahtallen good call! I will revert those now. 👍